---
title: "Testing pleiotropy vs. separate QTL in multiparental populations"
author: "Frederick Boehm, Brian Yandell, and Karl Broman"
date: "July 30, 2018"
bibliography: research.bib
output:
  xaringan::moon_reader:
    lib_dir: libs
    nature:
      highlightStyle: github
      highlightLines: true
      countIncrementalSlides: false
---

```{r setup, include=FALSE}
options(htmltools.dir.version = FALSE)
library(knitr)
```


```{r, load_refs, echo=FALSE, cache=FALSE}
library(RefManageR)
BibOptions(check.entries = FALSE, 
           bib.style = "authoryear", 
           cite.style = 'authoryear', 
           style = "markdown",
           hyperlink = FALSE, 
           dashed = FALSE, 
           longnamesfirst = FALSE,
           max.names = 2,
           bibpunct = c(" (", ") ", ",", ",", " and ", " "))
myBib <- ReadBib("./research.bib", check = FALSE)
```

# Introduction

- Experimentalists can now measure tens of thousands of phenotypes with RNA sequencing and mass spectrometry

- Multiparental populations enable high-resolution QTL mapping

- Together, high-dimensional phenotypes and multiparental populations can inform complex trait genetics

- New analysis tools, such as our test of pleiotropy vs. separate QTL, are needed 

---

# Multiparental populations

```{r, echo = FALSE, out.width = "700px"}
include_graphics("figs/ri8.png")
```








---

# Background

- `r Citet(myBib, "jiang1995multiple")` developed a pleiotropy vs. separate QTL test for F2 mice
    - Applies to two traits that map to a single genomic region 
    - Pleiotropy is the null hypothesis
    - Separate QTL is the alternative hypothesis
    - Perform a two-dimensional QTL scan 
    - Calculate likelihood ratio test statistic



---

# Challenges in multiparental populations

- Complex patterns of relatedness    
- Multiple founder lines (vs. exactly two founder lines in Jiang & Zeng)
- Test statistic calibration


---

# Solutions to challenges

- Multivariate random effects to account for relatedness    
- Fixed effect for each founder allele    
- Parametric bootstrap for test statistic calibration


---

# Test procedure

- Fit the model: $$Y = XB + G + E$$ for each ordered pair of markers
    - $G \sim MN(0, K, V_g)$   
    - $E \sim MN(0, I, V_e)$    
    - $X$ contains allele probabilities    
    - $B$ contains allele effects    
- Calculate likelihood for each model fit    
- Test statistic: $$- \log_{10} \frac{\max \text{likelihood under pleiotropy}}{\max \text{likelihood for separate QTL}}$$    
- Parametric bootstrap to get p-value



---

# Application to behavioral genetics data

- `r Citet(myBib, c("logan2013high", "recla2014precise"), .opts = list(max.names = 1))` genotyped and phenotyped 261 Diversity Outbred mice    
    - Identified *Hydin* as the chromosome 8 gene affecting "hot plate latency" at 57 cM
    - Identified chromosome 8 QTLs for "percent time in light" and "distance traveled in light" at 55 cM    
    - Motivated us to ask if *Hydin* also affects "percent time in light" and "distance traveled in light"    

```{r, echo = FALSE, out.width = "300px"}
include_graphics("figs/ccmice.png")
```


---

# Allele effects plots

- Suggest that "percent time in light" and "distance traveled in light" share a single QTL
- *Hydin* is distinct from that QTL

```{r, echo = FALSE, out.width = "400px"}
include_graphics("figs/coef_all3.svg")
```


---

# Profile LOD calculations

1. $LOD_{pvl}(\text{marker1}, \text{marker2}) =$ 
$\text{log-likelihood}(\text{marker1}, \text{marker2}) - \max \text{log-likelihood}(\text{marker}, \text{marker})$  

2. $\text{profile LOD}_1(\text{marker1}) = \max_{\text{marker2}}\text{LOD}_{pvl}(\text{marker1}, \text{marker2})$  

3. $LOD_{p}(\text{marker}) =  \text{log-likelihood}(\text{marker}, \text{marker}) - \max \text{log-likelihood}(\text{marker}, \text{marker})$



---

# Profile LOD plots for 3 pairwise analyses

```{r, echo = FALSE, out.width="400px"}
include_graphics("figs/all3.png", auto_pdf = FALSE)
```


---

# Three pairwise tests

- "percent time in light" & "hot plate latency": p = 0.109
- "distance traveled in light" & "hot plate latency": p = 0.108
- "percent time in light" & "distance traveled in light": p = 0.871


---

# Conclusions

- Evidence for two separate QTL affecting the 3 phenotypes
    - One QTL affects both "distance traveled in light" and "percent time in light"
    - Second QTL contains *Hydin* and affects "hot plate latency"


---

# Future directions

- Develop methods for analyzing multiple phenotypes in a single test  
- Accelerate calculations by writing costly calculations in C++  
- Enhance user interface for R package, qtl2pleio  



---

# Contact information

- frederick.boehm@gmail.com  
- https://github.com/fboehm/qtl2pleio  
- https://fboehm.us/  


---

# References

```{r, 'refs', results='asis', echo=FALSE}
PrintBibliography(myBib)
```

